src/main/java/tutu/task/Task.java

```diff
@@ -0,0 +1,49 @@
+package tutu.task;
+
+/** Represents a task. */
+public class Task {
+    /** Description of task */
+    protected String task;
+    /** Boolean to represent if the task is done */
+    protected boolean done;
```

Comment:
Consider adding "is" before the variable name to make it sound more like a Boolean variable, according to the Java coding standard.
```suggestion
    protected boolean isDone;
```